sawbuck_app/src/views/accept_offer_modal.js

```diff
@@ -186,6 +186,8 @@ const AcceptOfferModal = {
       modals.header('Accept Offer', vnode.attrs.cancelFn),
       modals.body(
         m('.container', [
+          m('.text-muted.mb-2',
+            `Enter how much ${vnode.state.holding.asset} you would like`),
```

Comment:
This leads to strange grammar for some offers. Would it be possible to just add the name of each asset over each dropdown, and have the name of the offer in the header?